libraries/eosiolib/core/eosio/print.hpp

```diff
@@ -2,6 +2,10 @@
  *  @file
  *  @copyright defined in eos/LICENSE
  */
+
+#ifndef __cdt_print_hpp_
+#define __cdt_print_hpp_
+
 #pragma once
```

Comment:
Why is `#pragma once` not enough?

Reply:
There is a note in the PR description explaining why. Def. would appreciate a better solution.

Reply:
This seems like a horrible hack to me. If we are going to add these ifdef guards then seems like we should add it to every header file. Better yet would be to fix the cmake or whatever is causing the file to come in from two diff locations. 

Probably should create a diff issue for this issue and work-around it for this PR some other way. Like maybe remove `#include "bitset.hpp"` from `datastream.hpp`.